config.json

```diff
@@ -180,6 +180,13 @@
         "text formatting",
         "preprocessor (x-macros in test)"
      ]
+  }, {
+     "difficulty": 1,
```

Comment:
Should this be a 1? Hello world is a 1 so I want to say this should be at least a 2 or 3.